db/migrate/20191030085853_add_import_source_to_document.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddImportSourceToDocument < ActiveRecord::Migration[5.2]
+  def change
+    add_column :documents, :import_source, :string
```

Comment:
I feel like this should be flagged as an enum on a document to indicate what is supported, as per https://github.com/alphagov/content-publisher/blob/master/app/models/status.rb#L18-L27

If we change this to be named `imported_from` we could use the [_prefix](https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html) option we can then get a boolean method on the document of `imported_from_whitehall?` for free